### ISSUE TITLE
Use `ifa_ifu` for associated_address in the Android

### ISSUE THREAD

**Issue by tkanakamalla** [mmastrac/getifaddrs#5]
The compilation of `getifaddrs` fails in the Android with the below error.
```
error[E0609]: no field `ifa_dstaddr` on type `&ifaddrs`
   --> /builds/gstreamer/cerbero/cerbero-sources/gst-plugins-rs/cargo-vendor/getifaddrs/src/lib.rs:286:34
    |
286 | ...                   .ifa_dstaddr
    |                        ^^^^^^^^^^^ unknown field
    |
help: a field with a similar name exists
    |
286 |                                 .ifa_addr
    |                                  ~~~~~~~~
For more information about this error, try `rustc --explain E0609`.
error: could not compile `getifaddrs` (lib) due to 1 previous error

```
Currently, it uses `ifa_dstaddr` for non-linux os but as per https://docs.rs/libc/latest/aarch64-linux-android/libc/struct.ifaddrs.html, but it should be `ifa_ifu` for android as well

**Comment by mmastrac** [comment]
Fixed in #6

**Comment by mmastrac** [comment]
Released as 0.1.5
